apps/lib/opt.c

```diff
@@ -738,6 +746,11 @@ int opt_verify(int opt, X509_VERIFY_PARAM *vpm)
             return 0;
         }
         break;
+    case OPT_V_EKU:
+        if ((i = opt_oid(opt_arg(), "Extended Key Usage")) == 0)
+            return 0;
+        X509_VERIFY_PARAM_set_eku(vpm, i);
```

Comment:
Variable names are cheap:
```suggestion
        if ((eku_nid = opt_oid(opt_arg(), "Extended Key Usage")) == NID_undef)
            return 0;
        X509_VERIFY_PARAM_set_eku(vpm, eku_nid);
```